compiler/rustc_middle/src/ty/context.rs

```diff
@@ -1277,11 +1277,10 @@ pub mod tls {
         rayon_core::tlv::get()
     }
 
+    /// A thread local variable that stores a pointer to the current `ImplicitCtxt`.
     #[cfg(not(parallel_compiler))]
-    thread_local! {
-        /// A thread local variable that stores a pointer to the current `ImplicitCtxt`.
-        static TLV: Cell<usize> = const { Cell::new(0) };
-    }
+    #[thread_local]
+    static TLV: Cell<usize> = const { Cell::new(0) };
```

Comment:
Why do we have an inline_const here? Couldn't that just call `Cell::new` directly?